pmdarima/tests/test_pipeline.py

```diff
@@ -203,6 +203,9 @@ def test_pipeline_predict_inverse_transform(pipeline, exog, inverse_transform,
 
     if return_conf_ints:
```

Comment:
I believe this test already covers what I have changed. I added more detailed tests on the shape and range of conf_ints. Let me know if you think any additional tests need to be added.